doc/faq.md

```diff
@@ -1,7 +0,0 @@
-**1. How to tell if my server instance is behind NAT?**
```

Comment:
This one too.

This PR (https://github.com/jitsi/jitsi-meet/pull/5960) should be reflected to https://jitsi.github.io/handbook/docs/faq